deploy/local/run_service.sh

```
@@ -0,0 +1,103 @@
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+CRISP_ROOT="$REPO_ROOT/examples/CRISP"
+SIGNAL_FILE="/tmp/crisp-dev-daemon-ready"
+
+SERVICE="${1:-}"
+
+die() { echo "[run_service] ERROR: $*" >&2; exit 1; }
+
+# ── Helpers ─────────────────────────────────────────────────────────────────
+wait_for_port() {
+    local host="${1:-localhost}" port="$2" label="${3:-$port}" max="${4:-120}"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)